lib/manager/components/version/FeedVersionDetails.js

```diff
@@ -119,86 +113,84 @@ export default class FeedVersionDetails extends Component<Props> {
-                // FIXME: uncomment out the below to prevent merges with non-future feeds.
-                // moment(summary.startDate).isAfter(moment().startOf('day')) &&
-                <VersionSelectorDropdown
-                  dropdownProps={{
-                    id: 'merge-versions-dropdown',
-                    disabled: isMergedServicePeriods,
-                    onSelect: this._handleMergeVersion
-                  }}
-                  title={<span><Icon type='code-fork' /> {mergeButtonLabel}</span>}
-                  itemFormatter={this._mergeItemFormatter}
-                  version={version}
-                  versions={feedSource.feedVersions}
-                />
-              }
-              <Button
-                disabled={
-                  !gtfsPlusValidation ||
-                  hasGtfsPlusBlockingIssue ||
-                  !gtfsPlusValidation.published ||
-                  !version.validationResult ||
-                  !version.validationResult.error_counts ||
-                  hasBlockingIssue ||
-                  isPublished ||
-                  processing ||
-                  !userCanManageFeed
-                }
-                bsStyle={isPublished ? 'success' : 'warning'}
-                onClick={this._onClickPublish}>
-                {isPublished
-                  ? <span><Icon type='check-circle' /> Published</span>
-                  : processing
-                    ? <span>Processing...</span>
-                    : <span>Publish to MTC</span>
-                }
-              </Button>
-            </ButtonToolbar>
-            : null
-          }
+        <h4 className='pull-left' style={{ marginBottom: '2px' }}>
           <span data-test-id='feed-version-validity'>
             <Icon type='calendar' />{' '}
-            Valid from {moment(summary.startDate).format(dateFormat)}{' '}
-            to {moment(summary.endDate).format(dateFormat)}
+            Feed validity dates
           </span>
-          {' '}
-          <VersionDateLabel version={version} />
-          {(hasBlockingIssue || hasGtfsPlusBlockingIssue) &&
-            isExtensionEnabled('mtc') &&
-            <div
-              className='text-danger'
-              style={{
-                position: 'absolute',
-                bottom: '1px',
-                right: '5px',
-                fontSize: 'x-small'
-              }}>
-              Cannot publish version because it has a{' '}
-              {hasGtfsPlusBlockingIssue ? 'GTFS+ ' : ''}
-              blocking issue.
-              (See{' '}
-              <Link
-                to={`/feed/${feedSource.id}/version/${version.version}/${hasGtfsPlusBlockingIssue ? 'gtfsplus' : 'issues'}`}>
-                {hasGtfsPlusBlockingIssue ? 'GTFS+' : 'validation'} issues
-              </Link>.)
-            </div>
-          }
         </h4>
+        <ButtonToolbar className='pull-right' style={{ marginTop: '2px' }}>
```

Comment:
Why is the ButtonToolbar included outside of the check for the MTC extension? If there is a reason for this it might need to be documented in a comment. If not, it seems like it would keep things tidier to revert to how this was.